server/views/pages/contacts/manage/contactConfirmation/basicDetails.njk

```diff
@@ -11,26 +34,23 @@
             text: "Name"
         },
         value: {
-            html: '',
-            attributes: {"data-qa": "confirm-name-value"}
+            html: '<span data-qa="confirm-name-value">' + contactFullNameRow | join(' ') + '</span>'
```

Comment:
What is the span adding here vs. using the attributes? I see it in a lot of places but it doesn't have any styling associated with it as far as I can tell

Reply:
Cypress does not find `attributes` added on `govukSummaryList` and fails. This is why I have added a span

I noticed that on `checkAnswers.njk` `classes: 'check-answers-dob-value'` is added for `govukSummaryList` which seems to be fine for cypress. 

I can try to change it to classes?

Reply:
replaced with classes attribute